android/src/main/java/com/oblador/keychain/supportBiometric/BiometricConstants.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2018 The Android Open Source Project
```

Comment:
I wonder if the copyright headers should be modified to indicate that the files have been modified for use in this library.

Reply:
I will double check, I am not sure.